deployment/localstack/terraform/provider.tf

```diff
@@ -6,3 +6,17 @@ terraform {
     }
   }
 }
+
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
+
+  access_key = "fake"
+  secret_key = "fake"
+
+  endpoints {
```

Comment:
i don't think localstack needs these
its probably passed in by default, I might be wrong though

Reply:
On my case it gives error if these `access_key` and `secret_key` is not provided.
![image](https://github.com/user-attachments/assets/24ee7eec-3032-4c9c-b1be-6ba533e6dc0a)

Reply:
Fair

Reply:
Although i was talking about the endpoints

Reply:
It wasn't clear about which block of code you asked about. As mostly `access_key` and `secret_key` was visible I thought you asked about that. 
BTW, `endpoints` block was introduced here due to multi region support on localstack. If I remove this block it throws some error:

```
 Error: Retrieving AWS account details: validating provider credentials: retrieving caller identity from STS: operation error STS: GetCallerIdentity, https response error StatusCode: 403, RequestID: 42f5bcf4-b92b-478c-bc53-234425607d00, api error InvalidClientTokenId: The security token included in the request is invalid.
│
│   with provider["registry.terraform.io/hashicorp/aws"].us-east-1,
│   on provider.tf line 10, in provider "aws":
│   10: provider "aws" {
```